### ISSUE TITLE
Feature Req: Push Notifications

### ISSUE THREAD

**Issue by profzelonka** [GreenMeteor/humhub-twa#7]
There's the outdated app https://github.com/evolved37/github_android_humhub which uses the old FirebaseInstanceIdService. It's since been replaced with the FirebaseMessagingService, not sure how similar it is so this may be a good reference.

The idea is to implement the firebase messaging service for push notifications. I'm not experienced to know if this guide is all that needs to be done? http://shubhank101.github.io/iOSAndroidChaosOverFlow/2016/06/Implementing-FCM-(-Firebase-Cloud-Messaging-)-in-Android-App

**Comment by ArchBlood** [comment]
First of all I'd like to say thank you for creating the issue, second I'd like to ask if you've tried using the following module beforehand?
https://github.com/humhub-contrib/fcm-push

**Comment by profzelonka** [comment]
Implementing the required SDK additions from Firebase into the build.gradle files creates errors and forces gradle version updates and other things, got it to build the app but it doesn't even open in itself anymore. How is fcm-push integrated with this android app? The integration is missing.

**Comment by ArchBlood** [comment]
> Implementing the required SDK additions from Firebase into the build.gradle files creates errors and forces gradle version updates and other things, got it to build the app but it doesn't even open in itself anymore. How is fcm-push integrated with this android app? The integration is missing.

The link that I shared is the module that is placed in `/protected/modules` and enabled on your HumHub instance side.

**Comment by ArchBlood** [comment]
As for your other issue when it comes to the implementation part, don't worry I'm currently working on an implementation for TWA.

**Comment by profzelonka** [comment]
> 

Right, but it does nothing without the app implementation else how does it know which mobile device to notify? 
Looking forward to your implementation! I've tried all I know and push notifications just don't happen. (Firebase's cloud notifications do tho, so we know that works off the bat.)
